lib/ex_unit/lib/ex_unit/doc_test.ex

```diff
@@ -185,9 +185,9 @@ defmodule ExUnit.DocTest do
 
   This macro is auto-imported with every `ExUnit.Case`.
   """
-  defmacro doctest(mod, opts \\ []) do
+  defmacro doctest mod, opts \\ [] do
```

Comment:
This should not happen on Elixir master. Could you please fetch the latest Elixir, run `make clean compile` and format again? Thank you.